commit df09d4ede93d20ed755f0d97778be359182fbcc7
Author: Kir Kolyshkin <kir@openvz.org>
Date:   Tue Mar 31 16:02:45 2009 +0400

    etc/init.d/vz-gentoo: add processing of multiple cron files
    
    This is an addition to commit 00a5a95c02c05e0070a1dd10b47c90b3715c9ff5
    for gentoo initscript. The idea is to be able to have multiple cronjob
    files managed by initscript.
    
    The only problem is there's no update_cron task for Gentoo initscript.
    
    http://bugzilla.openvz.org/599
    
    Signed-off-by: Kir Kolyshkin <kir@openvz.org>

diff --git a/etc/init.d/vz-gentoo.in b/etc/init.d/vz-gentoo.in
index 5050005..b1a1a59 100755
--- a/etc/init.d/vz-gentoo.in
+++ b/etc/init.d/vz-gentoo.in
@@ -58,8 +58,8 @@ setup_cron()
 {
 	[ -z "$SRC_CRONSCRIPT_DIR" ] && return
 	[ -d "$SRC_CRONSCRIPT_DIR" ] || return
-	install -m644 -o root -g root $SRC_CRONSCRIPT_DIR/vz \
-		$DST_CRONSCRIPT_DIR
+	cat $SRC_CRONSCRIPT_DIR/vz* > $DST_CRONSCRIPT_DIR/vz &&
+		chmod 644 $DST_CRONSCRIPT_DIR/vz
 }
 
 remove_cron()
@@ -70,8 +70,8 @@ remove_cron()
 # DO NOT EDIT THIS FILE!
 #
 # Contents of this file managed by /etc/init.d/vz script
-# Master copy is $SRC_CRONSCRIPT_DIR/vz.
-# Consult that file for documentation.
+# Master copy is in $SRC_CRONSCRIPT_DIR/vz* file(s).
+# Consult $SRC_CRONSCRIPT_DIR/vz* for documentation.
 EOF
 }
 
